crates/prql-compiler/src/semantic/resolver/snapshots/prql_compiler__semantic__resolver__test__frames_and_names-2.snap

```diff
@@ -10,12 +10,12 @@ columns:
       input_name: customers
       except: []
 inputs:
-  - id: 185
+  - id: 186
```

Comment:
FYI ignore these changes; it's an inconvenience that these change when we add an item to the stdlib...